app/src/main/java/com/example/caregiver/model/Task.java

```diff
@@ -16,6 +45,8 @@
     public String assignedStatus; /* true = task assigned to caregivee to do, false otherwise*/
     public String completionStatus; /* caregivee complete the tasks (complete), incomplete otherwise */
     public String room; /* room in which task was assigned */
+    public int timeCompleted = -1; /* (optional) time take to complete a task in seconds */
+    public long dateCompleted = -1; /* (optional) DATE when a task was completed in EpochTime */
 

```

Comment:
Could we make all these attributes private and add getters for them? We can also add setter methods as needed

Reply:
Let me merge this and create a separate issue. Since this code is also in other files. I also need to abstract out the getCaregiveeTask further because right now it still requires user to put in a firebaseRoomObject. My goal is to just have a function like List<Task> tasks = queryTask(caregiveeId) and it should do the job.